src/components/SelectNetworkAndAccount.svelte

```diff
@@ -100,12 +99,7 @@
     if (!selectedNetwork) return;
     isCustomNetwork = selectedNetwork.id === NetworkType.CUSTOM;
     if (!isCustomNetwork) {
-      if (selectedNetwork.endpoint && isValidURL(selectedNetwork.endpoint.toString())) {
-        const baseUrl =
-          $page.url.pathname === '/become-a-provider' ? $page.url.toString() : $page.url.origin.toString();
-        goto([baseUrl, selectedNetwork.pathName].join('/'));
```

Comment:
We don't ever use the network string that we were appending to the pathname so I removed